### PR TITLE
feat: add the accordion component

### PR DESCRIPTION
**Pull request by GusBaamonde**
**In a column in three and two:**

component-accordion

**Responsive:**

component-accordion-responsive

To see: http://cdn.fromdoppler.com/doppler-ui-library/build892/accordion-component.html

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

File: assets/scss/modules/_accordion.scss [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      padding: 24px 0 0 12px;
+      position: relative;
+
+      .dp-accordion-thumb {
+        margin: 0;
+        font-weight: bold;
+        cursor: pointer;
+        position: relative;
+        font-size: 16px;
+        color: #666;
```
Comment (cbernat) [inline assets/scss/modules/_accordion.scss:26]: You've missed a color variable here.

**Comment by cbernat** [comment]
Very nice! https://cdn.fromdoppler.com/doppler-ui-library/build892/accordion-component.html

**Comment by diusragus** [comment]
Great Gus! It's ok

**Comment by GusBaamonde** [comment]
test this please

**Pull request merged**